### DISCUSSION TITLE
MS OFFICE 2007

### DISCUSSION THREAD

**Discussion by crisgram** [GCuser99/SeleniumVBA#88] [Q&A] [answered]
I this working in MS office 2007? I got an error when adding add-in

**Comment by 6DiegoDiego9** [comment]
> I this working in MS office 2007? I got an error when adding add-in

Ho @crisgram, can you post a screenshot of the error?

**Comment by GCuser99** [comment, marked as answer]
Unfortunately [no support for Excel 2007 VBA6](https://github.com/GCuser99/SeleniumVBA/tree/main/dist#solution-summary). It would seem to be a straightforward endeavor to add that support, but am experienced enough to know that implementation is rarely ever as easy as it may seem at first thought... Sorry. Maybe someone else in the community can fork a try at that - I have no way to test on my systems.

**Comment by 6DiegoDiego9** [comment]
Oh yes, @GCuser99 is right. VBA7 was introduced with Excel 2010, and Excel 2007 uses VBA6.5. I didn't remember that, so @crisgram, it's not working primarily (or only?) because for supporting 64bit Excel it would require filling/dirtying the code with many conditional compiler constant expressions like "#If VBA7 then", which I personally think is no longer worth doing after so many years.

@crisgram, I see that Excel 2016 can be purchased online for as low as €0.49 (eg. https://www.eprice.it/pr/office-2016), are you tied to 2007 for some reasons?

**Comment by crisgram** [comment]
Thank you very much for the responses.
Yes it seems that 2007 is not supported.
I actually just updated it to 2016 and it works perfectly.
Thank you very much for this great tool!

**Comment by GCuser99** [comment]
Great @crisgram - don't forget to give us a Star! :-)
